gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
+                ignores: "",
```

Comment:
Can we not access the text area while it's not visible, making this storage redundant?